### PR TITLE
Luiz/fixed edit and csv format

### PR DESCRIPTION
**Pull request by luiztiago**

### CODE REVIEW

File: Controller/AbstractCoreCRUDController.php [hunk 1]
```diff
@@ -154,7 +154,7 @@ public function renderEdit(Request $request, array $criteria)
         $this->denyAccessUnlessGranted(EasyAdminVoterInterface::EDIT, $entity);
 
         $deleteForm = $this->createDeleteForm($request, $entity);
-        $editForm = $this->createForm($this->entityFormType, $entity);
+        $editForm = $this->createForm($this->entityFormType, $entity, array('validation_groups' => 'Default'));
```
Comment (johnmadrak) [inline Controller/AbstractCoreCRUDController.php]: @luiztiago - We can't add specific validation_groups in this bundle. If you need to add a validation group, please override renderEdit in your specific controller.

Comment (luiztiago) [inline Controller/AbstractCoreCRUDController.php]: agreed. removed! :)

File: Controller/AbstractCoreCRUDController.php [hunk 2]
```diff
@@ -94,7 +94,7 @@ public function renderCreate(Request $request)
             if ($form->isValid()) {
                 $this->entityManager->persist($entity);
                 $this->entityManager->flush();
-                $routeInfo = $this->getCreateRouteRedirect($request, $entity);
+                $routeInfo = $this->getCreateRouteRedirect($request, $entity, $this->getCreateFormOptions());
```
Comment (johnmadrak) [inline Controller/AbstractCoreCRUDController.php]: @luiztiago - Did you intentionally put this here instead of line 90?

Comment (luiztiago) [inline Controller/AbstractCoreCRUDController.php]: @johnmadrak nops, good catch! fixed!

**Pull request merged**